README.md

```diff
@@ -85,6 +89,12 @@ Use the following settings to update cell execution time for a notification and
     // The minimum execution time to send out notification for a particular notebook cell (in seconds).
     "minimum_cell_execution_time": 60,
 
+    // Notification Methods
+    // Methods how to notificate messages. Select from 'browser' or 'ntfy'
```

Comment:
Could you update the description to this? I just to make sure the users understand that they can choose more than one.
```suggestion
    // Option to send a notification with the specified method(s). The available options are 'browser' and 'ntfy'.
```